Controllers/CartController.cs

```diff
@@ -117,6 +118,41 @@ public CartController(BangazonContext ctx)
                      new { controller = "Products", action = "Detail", Id = id } ) );
         }
 
+        public async Task<IActionResult> DeleteLineItem([FromRoute]int id)
+        {
+            if (SessionHelper.ActiveUser == null)
+            {
+                SessionHelper.ActiveUser = ActiveUser.Instance.User.UserId;
+            }
+
+            Order OpenOrder = await(
+                from order in context.Order
+                where order.UserId == SessionHelper.ActiveUser && order.DateCompleted == null
```

Comment:
This should reference Active User like so:
`where order.UserId == ActiveUser.Instance.User.UserId && order.DateCompleted == null`